version.txt

```diff
@@ -52,4 +52,4 @@ veg_plots_sdl.json,1.1.1
 vertnet.py,1.3.0
 wine_composition.json,1.1.1
 wine_quality.json,1.1.1
-wood_density.py,1.2.1
+wood_density.py,1.2.1
```

Comment:
I've noticed some flipping back and forth between having a new line and not having one at the end of version.txt, which is what we're seeing here. I suspect it's an OS difference in precisely how `version.py` is running or maybe some folks are opening `version.txt` in their editors which are adding a newline.